rules/test.bzl

```diff
@@ -104,7 +118,7 @@ def _make_test(name, test_rule, **kwargs):
     Helper to create an individual test
     """
     runner = kwargs.pop("runner", None) or _DEFAULT_APPLE_TEST_RUNNER
-    test_attrs = {k: v for (k, v) in kwargs.items() if k not in _APPLE_BUNDLE_ATTRS}
+    test_attrs = {k: v for (k, v) in kwargs.items() if (k not in _APPLE_BUNDLE_ATTRS) or (k in _ALWAYS_AVAILABLE_ATTRS)}
```

Comment:
when downstream project point to this commit, probably worth printing out what is dropped before VS after, to see if anything other than `features` are dropped also

Reply:
Do you mean adding a `print` here?

Reply:
yeah but do it temp. during the testing